maxim/tests/test_logger_azure_openai.py

```
@@ -8,35 +8,46 @@
 
 from .. import Config, Maxim
 from ..logger import GenerationConfig, LoggerConfig, TraceConfig
+from ..tests.mock_writer import inject_mock_writer
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)